pkg/kubeapiserver/options/authentication.go

```diff
@@ -260,6 +276,9 @@ func (s *BuiltInAuthenticationOptions) AddFlags(fs *pflag.FlagSet) {
 		fs.StringSliceVar(&s.ServiceAccounts.APIAudiences, "service-account-api-audiences", s.ServiceAccounts.APIAudiences, ""+
 			"Identifiers of the API. The service account token authenticator will validate that "+
 			"tokens used against the API are bound to at least one of these audiences.")
+
+		fs.Int64Var(&s.ServiceAccounts.MaxExpTime, "service-account-max-exp-time", s.ServiceAccounts.MaxExpTime, ""+
```

Comment:
can you call this `service-account-token-max-expiration` and give it a type of pflag.Duration:

https://godoc.org/github.com/spf13/pflag#FlagSet.Duration